pkg/operator/starter.go

```diff
@@ -98,10 +98,31 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	); err != nil {
 		return err
 	}
+	if err := resourceSyncController.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorNamespace, Name: "initial-client-ca"},
+		resourcesynccontroller.ResourceLocation{Namespace: userSpecifiedGlobalConfigNamespace, Name: "initial-client-ca"},
+	); err != nil {
+		return err
+	}
+	// this ca bundle contains certs used to sign CSRs (kubelet serving and client certificates)
+	if err := resourceSyncController.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorNamespace, Name: "csr-controller-ca"},
+		resourcesynccontroller.ResourceLocation{Namespace: machineSpecifiedGlobalConfigNamespace, Name: "csr-controller-ca"},
+	); err != nil {
+		return err
+	}
+	// this ca bundle contains certs used by the kube-apiserver to verify client certs
+	if err := resourceSyncController.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: machineSpecifiedGlobalConfigNamespace, Name: "kube-apiserver-client-ca"},
```

Comment:
This is an outbound sync that we will consume in the openshift-apiserver operator so that we never flap on different static pods restarting with different certs.  We will always use what this operator says is good.